src/actors/runtime/indices/indices.go

```diff
@@ -1,20 +1,35 @@
 package indices
 
 import (
+	"math/big"
+
 	abi "github.com/filecoin-project/specs/actors/abi"
 	actor_util "github.com/filecoin-project/specs/actors/util"
-	"math/big"
 )
 
 var PARAM_FINISH = actor_util.PARAM_FINISH
 
-func StorageDeal_ProviderInitTimedOutSlashAmount(providerCollateral abi.TokenAmount) abi.TokenAmount {
-	// placeholder
-	PARAM_FINISH()
-	return providerCollateral
-}
-
-func (inds *Indices_I) StorageDeal_DurationBounds(
+// Data in Indices are populated at instantiation with data from the state tree
+// Indices itself has no state tree or access to the runtime
+// it is a passive data structure that allows for convenience access to network indices
+// and pure functions in implementing economic policies given states
+type Indices struct {
```

Comment:
Please define an interface with the name `Indices` and make that the return value from the runtime. It should probably have all the methods currently defined for this struct.

 Then rename this struct to implement the interface.

Reply:
done.